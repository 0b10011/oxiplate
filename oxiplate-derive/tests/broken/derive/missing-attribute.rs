use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
struct Data;

fn main() {}
