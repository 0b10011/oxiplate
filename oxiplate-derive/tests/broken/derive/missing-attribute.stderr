error: Expected an attribute named `oxiplate_inline` or `oxiplate` to specify the template:
       External: #[oxiplate = "path/to/template/from/templates/directory.html.oxip"]
       Internal: #[oxiplate_inline(html: "{{ your_var }}")]
 --> tests/broken/derive/missing-attribute.rs:4:8
  |
4 | struct Data;
  |        ^^^^
